.eslintrc

```diff
@@ -1,9 +1,6 @@
 {
     "extends": [
-        "spt",
-        "spt/env-node"
-    ],
-    "globals": {
```

Comment:
This is a breaking change, but as the entire commit is, it's not an extra issue.  Should probably be mentioned in the changelog, tho.